packages/frontend/src/components/helpers/PseudoListItem.tsx

```diff
@@ -117,13 +117,13 @@ export const PseudoListItemAddContact = ({
   const settingsStore = useSettingsStore()[0]
   const isChatmail = settingsStore?.settings.is_chatmail === '1'
 
+  if (isChatmail) return null
```

Comment:
This approach is IMO not the best semantically, might cause bugs in the future. Namely, `contactsAndExtraItems.length` is calculated to account for the presence of this element, but it's not present.

https://github.com/deltachat/deltachat-desktop/blob/a36f56ff9c7600ef380f90db33dcae54cb48f08b/packages/frontend/src/components/dialogs/CreateChat/index.tsx#L345

https://github.com/deltachat/deltachat-desktop/blob/a36f56ff9c7600ef380f90db33dcae54cb48f08b/packages/frontend/src/components/dialogs/CreateChat/index.tsx#L212-L226

It's better to modify `contactsAndExtraItems`, in case of the `CreateChat` component. For other components where `PseudoListItemAddContact` is used, it's probably the same.

Reply:
Yes I will address this in the follow up (that's why the branch is named hide-add-manually-step1

But this change makes sure the PseudoListItemAddContact is never shown on chatmail accounts.

We need to pass a parameter to the AddMemberDialog to decide wether the add contact manually should be shown etc.

Reply:
> But this change makes sure the PseudoListItemAddContact is never shown on chatmail accounts.

Yes, I think keeping the condition in two places, inside and outside, is ok.